Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'bootsnap', '>= 1.4.2', require: false
 gem 'jbuilder', '~> 2.7'
 gem 'mysql2', '>= 0.5.2'
 gem 'puma', '~> 3.11'
+gem 'rack-cors'
```

Comment:
Its a good practice to specify gem versions in the Gemfile. 
Also we should be pushing the gemfile.lock too. The gemfile.lock file records the exact version of gems being installed and thus helps in mitigating any dependency issues that may arise when working on different machine. 

https://bundler.io/guides/using_bundler_in_applications.html#gemfilelock

Reply:
ok. I checked cors version in Gemfile.lock which is "rack-cors (1.1.0)". So shall I update the Gemfile with this version?

Reply:
The Gemfile.lock is in gitignore so shall we remove it from there or it should be in gitignore itself?

Reply:
Yes, please edit the git-ignore file and push the changes. The gemfile can be updated as follows:
```ruby
gem 'rack-cors', '~> 1.1.0'
```

Reply:
ok. I will do the changes soon.